x-pack/test/functional/apps/infra/hosts_view.ts

```diff
@@ -362,6 +404,17 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
           it('should render logs tab', async () => {
             await pageObjects.assetDetails.logsExists();
           });
+
+          it('preserves search term between page reloads', async () => {
+            const searchInput = await pageObjects.assetDetails.getLogsSearchField();
+
+            expect(await searchInput.getAttribute('value')).to.be('');
+
+            await searchInput.type('test');
+            await refreshPageWithDelay();
+
+            expect(await searchInput.getAttribute('value')).to.be('test');
```

Comment:
Nit: After the tests checking the input values and changing the input is a good idea to clear the input values added - In this case, there shouldn't be an issue but if you have other tests checking for values that are not present because of the new filter added it will fail.

Reply:
good point, thank you!